src/import/FSHImporter.ts

```diff
@@ -144,7 +144,7 @@ export class FSHImporter extends FSHVisitor {
   paramRuleSets: Map<string, ParamRuleSet>;
   private topLevelParse: boolean;
   private pathContext: string[][];
-  private baseIndent: number;
+  private baseIndent = 1;
```

Comment:
Do we still need `baseIndent` and all of its processing? As I remember this was only used to determine relative indent, so why is it needed if there are no more relative indents?

Reply:
I suppose we don't, I'll remove this more thoroughly.

Reply:
Commit `94220bf` removes the `baseIndent` class variable and some of the references to it, but I did think there was value in having the default start column saved as a constant variable so I added that instead.